### DISCUSSION TITLE
Gaia-X Compliant Self-Descriptions in EDC / MVD

### DISCUSSION THREAD

**Discussion by kaimeinke** [eclipse-edc/.github#36] [Q&A] [answered]
Dear EDC-Community,

are there any Gaia-X compliant Self-Descriptions and Examples available for the EDC and/or can you point me to the repository where this feature and Self-Descriptions are expected to be created?

I am aware of https://github.com/eclipse-edc/IdentityHub/issues, but could not find much there.
I am also aware of https://github.com/eclipse-edc/MinimumViableDataspace/tree/main/system-tests/resources/webdid, but could not find any verifiable Self-Descriptions there as well.

Thank you very much.

Kai

**Comment by bscholtes1A** [comment, marked as answer]
Hello @kaimeinke,

Here is a quick sum-up of what we have today and what are the ongoing works on the Gaia-X topic:

 - today our https://github.com/eclipse-edc/IdentityHub exposes a REST endpoint returning the self-description that you should have previously created, signed and validated against the compliance API (all this steps being done manually), which is what we have been using so far during the previous Gaia-X Hackathon 
 - we are migrating all EDC extensions and services related to Gaia-X into this recently-created [Trust Framework repository](https://github.com/eclipse-edc/TrustFrameworkAdoption), which means that the previous Identity Hub Self-Description endpoint will be decomissioned. 
 - it will be replaced by a brand new extension in the Trust Framework repository that will enable to dynamically generated a valid Self-Description based on an set of claims passed in input (set of claims that might for example be created using the GXFS wizard). This extension will also sign the claims passed in input and store the associated VC into the store of the Identity Hub. 
 - also in this Trust Framework repository we already provide an extension enabling to define access policies based on the claims structure such as defined in the [Ontology document for data exchanges](https://gaia-x.gitlab.io/technical-committee/federation-services/data-exchange/dewg/#ontologies-for-data-exchange)
